server.js

```diff
@@ -40,125 +42,189 @@ nunjucks.setup({
   noCache: true
 }, app);
 
+
 // require core and custom filters, merges to one object
 // and then add the methods to nunjucks env obj
+
 nunjucks.ready(function(nj) {
   var coreFilters = require(__dirname + '/lib/core_filters.js')(nj),
     customFilters = require(__dirname + '/app/filters.js')(nj),
     filters = Object.assign(coreFilters, customFilters);
   Object.keys(filters).forEach(function(filterName) {
     nj.addFilter(filterName, filters[filterName]);
   });
-});
 
-// Middleware to serve static assets
-app.use('/public', express.static(__dirname + '/public'));
-app.use('/public', express.static(__dirname + '/govuk_modules/govuk_template/assets'));
-app.use('/public', express.static(__dirname + '/govuk_modules/govuk_frontend_toolkit'));
-app.use('/public/images/icons', express.static(__dirname + '/govuk_modules/govuk_frontend_toolkit/images'));
-
-// Elements refers to icon folder instead of images folder
-app.use(favicon(path.join(__dirname, 'govuk_modules', 'govuk_template', 'assets', 'images','favicon.ico')));
-
-// Support for parsing data in POSTs
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({
-  extended: true
-}));
-
-// send assetPath to all views
-app.use(function (req, res, next) {
-  res.locals.asset_path="/public/";
-  next();
-});
+  app.use(function(req,res,next){
 
-// Add variables that are available in all views
-app.use(function (req, res, next) {
-  res.locals.serviceName=config.serviceName;
-  res.locals.cookieText=config.cookieText;
-  res.locals.releaseVersion="v" + releaseVersion;
-  next();
-});
+    // add nunjucks function to get values, needs to be here as they need access to req.session
 
-// Force HTTPs on production connections
-if (env === 'production' && useHttps === 'true'){
-  app.use(utils.forceHttps);
-}
+    nj.addGlobal("checked",function(name, value){
```

Comment:
This `addGlobal` function seems rather spaced out compared to the rest of the stuff in this page - does it need so many linebreaks?